### ISSUE TITLE
bug: `v0.20.5` windows `.zip` file detected as `Trojan:Sript/Phonzy.B!ml` by Microsoft Defender

### ISSUE THREAD

**Issue by igotinfected** [eza-community/eza#1205]
Referring to https://github.com/eza-community/eza/releases/download/v0.20.5/eza.exe_x86_64-pc-windows-gnu.zip
The `v0.20.4` download doesn't trigger this.

![image](https://github.com/user-attachments/assets/c489d93e-374e-4292-a3d8-7df52a6c6fbf)

**Comment by cafkafk** [comment]
This is a duplicate of  #419. Sadly this is out of our control, windows defender keeps detecting some releases as trojans, because their virus detection isn't... great.

If you want this unblocked, you can submit the false positive to the windows defender team.
